app/javascript/components/widget-controller.jsx

```diff
@@ -6,15 +6,12 @@ import {
   WidgetDisplay,
   LoadingDisplay,
   DisconnectedDisplay,
-  getWidgetDisplay,
 } from '@components/widget-display';
 import WidgetTransitionControls from '@components/widget-transition-controls';
 
 const getWidgets = gql`
-  ${getWidgetDisplay}
   query getWidgets($id: Int!, $cityName: String!) {
     location(cityName: $cityName) {
-      ...WidgetDisplay
```

Comment:
Why was the fragment deleted from the fragment?

Reply:
I probably accepted the wrong changes during a rebase? :confused: Thanks for pointing that out, I'll fix it in the next commit